Summary Table/Summary Table Dev Flextable.Rmd

```diff
@@ -0,0 +1,398 @@
+---
+title: ''
+output: 
+  word_document:
+    reference_docx: "//conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/Summary Table/ST_ref_doc.docx"
+---
+;
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE, comment = NA)
+# Lines below for testing only:
+# LOCALITY <- "Falkirk West"
+# LOCALITY <- "Stirling City with the Eastern Villages Bridge of Allan and Dunblane"
+# LOCALITY <- "Mid-Argyll, Kintyre and Islay"
+# LOCALITY <- "City of Dunfermline"
+# LOCALITY <- "Barra"
+# LOCALITY <- "Whalsay and Skerries"
+# LOCALITY <- "Inverclyde East"
+ 
+# setwd("~/list-localities-profiles/")
+
+## Project file path
+ lp_path <- "/conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/"
+ 
+# Source in global functions/themes script
+ source("./Master RMarkdown Document & Render Code/Global Script.R")
+ 
+# # demographics
+ source("./Demographics/1. Demographics - Population.R")
+ source("./Demographics/2. Demographics - SIMD.R")
+#
+# # housing
+ source("./Households/Households Code.R")
+#
+# # general health
+ source("./General Health/3. General Health Outputs.R")
+#
+# # lifestyle & risk factors
+ source("./Lifestyle & Risk Factors/2. Lifestyle & Risk Factors Outputs.R")
+#
+# # unscheduled care
+ source("./Unscheduled Care/2. Unscheduled Care outputs.R")
+
+library(tidyverse)
+library(knitr)
+library(gridExtra)
+library(grid)
+library(kableExtra)
+library(dplyr)
+library(flextable)
+library(officer)
+library(officedown)
+
+# detach(package:tidylog, unload = TRUE)
+```
+
+
+```{r echo=FALSE}
+# Set locality name object (with automatic wrapping)
+loc_name <- paste0(LOCALITY, "\nLocality") %>%
+  strwrap(width = case_when(n_loc < 4 ~ 20, n_loc == 4 ~ 15, n_loc > 4 ~ 8), simplify = FALSE)
+loc_name <- sapply(loc_name, paste, collapse = "\n")
```

Comment:
`str_wrap` is from stringr (loaded with the tidyverse), it uses a better algorithm (https://en.wikipedia.org/wiki/Knuth%E2%80%93Plass_line-breaking_algorithm) and also returns a character vector like we want by default.
```suggestion
loc_name <- paste(LOCALITY, "Locality", sep = "\n") %>%
  str_wrap(width = case_when(n_loc < 4 ~ 20, n_loc == 4 ~ 15, n_loc > 4 ~ 8))
```